pkgs/servers/sql/postgresql/default.nix

```diff
@@ -87,7 +99,6 @@ let
       "--with-libxml"
       "--with-icu"
       "--sysconfdir=/etc"
-      "--libdir=$(lib)/lib"
```

Comment:
I'm probably missing something, but how are `.so` files moved to the `lib` output now?

Reply:
We previously passed `setOutputFlags = false;` so we had to set it explicitly. Now that I removed it, multiple-outputs hook passes this for us.